microverse-trials-leo-joel-main/fcc-test-suite-template-master/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
Please, update the title from your site because this is one is from the template.